tests/access_test.go

```diff
@@ -53,6 +54,51 @@ var _ = Describe("[rfe_id:500][crit:high][vendor:cnv-qe@redhat.com][level:compon
 		tests.BeforeTestCleanup()
 	})
 
+	Describe("With the kubevirt-controller service account", func() {
+		table.DescribeTable("should verify permissions on resources are correct for kubevirt-controller", func(resource string) {
+
+			controllerVerbs := make(map[string]string)
+			controllerVerbs["get"] = "yes"
+			controllerVerbs["list"] = "yes"
+			controllerVerbs["watch"] = "yes"
+			controllerVerbs["delete"] = "no"
+			controllerVerbs["create"] = "no"
+			controllerVerbs["update"] = "no"
+			controllerVerbs["patch"] = "yes"
+			controllerVerbs["deleteCollection"] = "no"
+
+			// Adjustment for the rule associated with kubevirt, vmipresets, vmim
+			switch resource {
+			case "kubevirt":
+				controllerVerbs["patch"] = "no"
+			case "virtualmachineinstancepresets":
+				controllerVerbs["get"] = "no"
+				controllerVerbs["patch"] = "no"
+			case "virtualmachineinstancemigrations":
+				controllerVerbs["create"] = "yes"
+			}
+
+			namespace := tests.NamespaceKubevirt // Is there an exported var defines kubevirt ns ?
+			verbs := []string{"get", "list", "watch", "delete", "create", "update", "patch", "deletecollection"}
+
+			for _, verb := range verbs {
+				By(fmt.Sprintf("verifying kubevirt-controller for verb %s", verb))
+				expectedRes, _ := controllerVerbs[verb]
+				as := fmt.Sprintf("system:serviceaccount:%s:%s", namespace, controller)
+				result, _, _ := tests.RunCommand(k8sClient, "auth", "can-i", "--as", as, verb, resource)
+				Expect(result).To(ContainSubstring(expectedRes))
+			}
+		},
+			// The test id is temporary, appearing unique
+			table.Entry("[test_id:626]given a kv", "kubevirt"),
+			table.Entry("[test_id:627]given a vm", "virtualmachines"),
+			table.Entry("[test_id:626]given a vmi", "virtualmachineinstances"),
+			table.Entry("[test_id:629][crit:low]given a vmi replica set", "virtualmachineinstancereplicasets"),
```

Comment:
You have a duplicated test_id. Where did you get the test id's from?

Reply:
I just put them in, did a grep on them in the code tree, found no duplicate. I did not know where `test_id` generally came from. I would be happy to take suggestions on the `test_id`: either to remove them in the field or change to more aligned. 

I will definitely remove fix the duplicate in the code.